biogtr/visualize.py

```diff
@@ -0,0 +1,292 @@
+"""Helper functions for visualizing tracking."""
+from scipy.interpolate import interp1d
+from copy import deepcopy
+from tqdm import tqdm
+from matplotlib import pyplot as plt
+from omegaconf import DictConfig
+
+import seaborn as sns
+import imageio
+import hydra
+import pandas as pd
+import numpy as np
+import cv2
+import imageio
+
+
+palette = sns.color_palette("tab10")
+
+
+def fill_missing(Y: np.ndarray, kind: str = "linear") -> np.ndarray:
+    """Fills missing values independently along each dimension after the first.
+
+    Args:
+        Y: the array for which to fill missing value
```

Comment:
Maybe we have more clear variable names? Might make sense to move this to data utils - in case we ever want to use for training?